src/com/carolinarollergirls/scoreboard/core/impl/SkaterImpl.java

```diff
@@ -113,16 +113,23 @@ public ValueWithId create(AddRemoveProperty prop, String id) {
     }
     @Override
     protected void itemAdded(AddRemoveProperty prop, ValueWithId item) {
-        if (prop == NChild.PENALTY && FO_EXP_ID.equals(((Penalty)item).getProviderId())) {
-            updateEligibility();
-        } else if (prop == NChild.PENALTY && !((Penalty)item).isServed() && getRole() == Role.JAMMER
-                && !getCurrentFielding().getTeamJam().getOtherTeam().isLead()) {
-            getTeam().set(Team.Value.LOST, true);
-        } else if (prop == Child.FIELDING && ((Fielding)item).isCurrent()) {
-            set(Value.CURRENT_FIELDING, item, Flag.INTERNAL);
-        } else if (prop == Child.FIELDING && team.hasFieldingAdvancePending() &&
-                ((Fielding)item).getTeamJam().isRunningOrEnded()) {
-            setRole(((Fielding)item).getCurrentRole());
+        if (prop == NChild.PENALTY) {
+            Penalty p = (Penalty)item;
+            if (FO_EXP_ID.equals(p.getProviderId())) {
+                updateEligibility();
+            } else if (!p.isServed() && getRole() == Role.JAMMER && getCurrentFielding() != null
```

Comment:
This is making it not work for Jammers

Reply:
This should not fire between jams - fixed.